cmd/registry/cmd/export/export.go

```diff
@@ -65,11 +65,11 @@ func Command() *cobra.Command {
 			}
 			// Visit the selected resources.
 			if err = visitor.Visit(ctx, v, visitor.VisitorOptions{
-				RegistryClient:          registryClient,
-				AdminClient:             adminClient,
-				Pattern:                 pattern,
-				Filter:                  filter,
-				AllowUnavailableProject: true,
+				RegistryClient:  registryClient,
+				AdminClient:     adminClient,
+				Pattern:         pattern,
+				Filter:          filter,
+				ImplicitProject: &rpc.Project{Name: "Implicit"},
```

Comment:
I would expect "Implicit" to fail name validation. I think we know the name that we want here, so I think we could use that instead.

Reply:
oh, actually we don't, because we don't know what "pattern" is.

Reply:
How about leaving it as &rpc.Project{} and filling in the name when we know it? (for gets, and I guess lists could use the value that we pass in)